examples/notify-turn-hook.py

```
@@ -0,0 +1,248 @@
+        return ""
+
+    prompt = f"""Output ONLY a single short sentence (max 15 words) summarizing what was done. No questions, no explanations, no preamble. Just the summary sentence.
+
+User asked: {last_user_msg[:200]}
+
+Assistant did: {assistant_msgs[:1500]}"""
+
+    try:
+        LOCKFILE.touch()
+        result = subprocess.run(
+            ["claude", "--print", "--model", "haiku"],
```

Ruff (0.14.6)

Starting a process with a partial executable path

(S607)